To: vim_dev@googlegroups.com
Subject: Patch 7.3.190
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.190
Problem:    When there is a "containedin" syntax argument highlighting may be
	    wrong. (Radek)
Solution:   Reset current_next_list. (Ben Schmidt)
Files:	    src/syntax.c


*** ../vim-7.3.189/src/syntax.c	2011-04-11 16:56:29.000000000 +0200
--- src/syntax.c	2011-05-19 12:02:43.000000000 +0200
***************
*** 2566,2571 ****
--- 2566,2574 ----
  #endif
  		update_si_attr(current_state.ga_len - 1);
  
+ 		/* nextgroup= should not match in the end pattern */
+ 		current_next_list = NULL;
+ 
  		/* what matches next may be different now, clear it */
  		next_match_idx = 0;
  		next_match_col = MAXCOL;
*** ../vim-7.3.189/src/version.c	2011-05-10 17:29:28.000000000 +0200
--- src/version.c	2011-05-19 12:13:28.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     190,
  /**/

-- 
From "know your smileys":
 :'-D	Laughing so much that they're crying

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
